UPGRADE-3.0.md

```diff
@@ -15,7 +15,7 @@
 
 ### Authentication methods
 
-* `Github\Client::AUTH_URL_TOKEN` use `Github\Client::AUTH_ACCESS_TOKEN` instead.
-* `Github\Client::AUTH_URL_CLIENT_ID` use `Github\Client::AUTH_CLIENT_ID` instead.
-* `Github\Client::AUTH_HTTP_TOKEN` use `Github\Client::AUTH_ACCESS_TOKEN` instead.
-* `Github\Client::AUTH_HTTP_PASSWORD` use `Github\Client::AUTH_ACCESS_TOKEN` instead.
+* `Github\Client::AUTH_ACCESS_TOKEN`  use `Github\AuthMethod::AUTH_ACCESS_TOKEN` instead.
```

Comment:
Please revert these changed as this is the upgrade file for v2 -> v3. Add your upgrade notes to the `UPGRADE-4.0.md` file.